packages/xo-server/src/xapi/index.js

```diff
@@ -1122,14 +1122,32 @@ export default class Xapi extends XapiBase {
       return defaultSr.$ref
     })
 
+    const hostPbds = new Set(host.PBDs)
+    const connectedSrs = new Map()
+    const isConnectedSr = sr => {
+      let isConnected = connectedSrs.get(sr.$ref)
+      if (isConnected === undefined) {
+        isConnected = sr.PBDs.find(ref => hostPbds.has(ref))
```

Comment:
If you don't need to use the PBD later, make it an actual boolean.

/cc @julien-f

Reply:
I don't understand, make what a boolean?

Reply:
The `isConnected` variable. Right now it's a PBD ref and I don't see why it needs to be a PBD ref.

Reply:
Indeed, it's a mistake on my part, it should be `Array#some` instead:
```suggestion
        isConnected = sr.PBDs.some(ref => hostPbds.has(ref))
```

Reply:
It will break the line bellow.